### PR TITLE
DDF-3636: Update documentation to reflect application/karaf feature decoupling

### PR DESCRIPTION
**Pull request by tbatie**
#### What does this PR do?
See PR title

#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@emanns95 
@vinamartin 

#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@rzwiefel 
@ricklarsen - Documentation

#### How should this be tested? (List steps with links to updated documentation)
CI passes

#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_appReferences/mg-admin.adoc [hunk 1]
```diff
@@ -6,12 +6,9 @@
 
 == {title} Application Reference
 
-The ${ddf-admin} Application contains components that are integral for the installation and configuration of ${branding} applications.
+The ${ddf-admin} Application contains components that are integral for the configuration of ${branding} applications.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_appReferences/mg-admin.adoc]: ✏️ I think we might as well lose this extra space and combine to one paragraph.

File: distribution/docs/src/main/resources/content/_appReferences/mg-admin.adoc [hunk 2]
```diff
@@ -6,12 +6,9 @@
...
 
-It contains various services and interfaces that allow administrators more control over their systems and enhances administrative capabilities when installing and managing ${branding}.
-
-The ${ddf-admin} application contains an application service that handles all operations that are performed on applications.
-This includes adding, removing, starting, stopping, and showing status.
+It contains various services and interfaces that allow administrators more control over their systems and enhances administrative capabilities.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_appReferences/mg-admin.adoc]: ✏️ I don't think "more control" makes sense any more now, as there's only on admin UI. Can you cut it?

File: distribution/docs/src/main/resources/content/_appReferences/mg-broker-app.adoc [hunk 3]
```diff
@@ -19,9 +19,9 @@ The ${message-broker} app is uninstalled by default and requires manual installa
 
 To install:
 
-. Navigate to the *${admin-console}*.
-. Select *Manage Applications*.
-. Select the *${message-broker}* application.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_appReferences/mg-broker-app.adoc]: Always start with Navigate to the ${admin-console}, then Select *System* tab.

File: distribution/docs/src/main/resources/content/_appReferences/mg-catalog.adoc [hunk 4]
```diff
@@ -14,7 +14,7 @@ At the core of the Catalog functionality is the Catalog Framework, which routes
 
 === ${ddf-catalog} Application Prerequisites
 
-To use the ${ddf-catalog} Application, the following applications/features must be installed:
+To use the ${ddf-catalog} Application, the following `applications/features must be installed:
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_appReferences/mg-catalog.adoc]: stray back tick character

File: distribution/docs/src/main/resources/content/_appReferences/mg-geowebcache.adoc [hunk 5]
```diff
@@ -28,10 +28,9 @@ The ${ddf-geowebcache} application is *not* installed by default.
 
 To install:
 
-. Navigate to the ${admin-console}.
-. Select *Manage*.
-. Select the ${ddf-geowebcache} application *Start* button. The application will move to *Active Applications* on startup.
-. Select *Back* in ${admin-console}.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_appReferences/mg-geowebcache.adoc]: same comment re: starting at admin console

File: distribution/docs/src/main/resources/content/_appReferences/mg-registry.adoc [hunk 6]
```diff
@@ -24,10 +24,9 @@ ${ddf-registry} is not installed by default with a standard installation.
 
 To install:
 
-. Navigate to the *${admin-console}*.
-. Click the *Manage* applications button.
-. Click the *Install* icon.
-. ${ddf-registry} will move to *Active Applications* upon startup.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_appReferences/mg-registry.adoc]: here too on admin console

File: distribution/docs/src/main/resources/content/_appReferences/mg-resource-mgmt.adoc [hunk 7]
```diff
@@ -34,9 +34,7 @@ The ${ddf-resource-management} application is not installed by default with a st
 
 To Install:
 
-. Navigate to ${admin-console}.
-. Select *Manage* applications.
-. Select *${ddf-resource-management}* application.
-. Select *Start*.
-. ${ddf-resource-management} application moves to *Active Applications* upon start.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_appReferences/mg-resource-mgmt.adoc]: same

File: distribution/docs/src/main/resources/content/_configuring/admin-console-tutorial.adoc [hunk 8]
```diff
@@ -8,36 +8,11 @@
...
 .System Settings Tab
-The configuration and features installed can be viewed and edited from the System tab as well; however, it is recommended that configuration be managed from the applications tab.
-
-[IMPORTANT]
-====
-In general, applications should be managed via the applications tab.
-Configuration via this page could result in an unstable system.
-Proceed with caution!
-====
+The configuration and features installed can be viewed and edited from the System tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_configuring/admin-console-tutorial.adoc]: Maybe bold *System* tab, and and add *System* tab `of the ${admin-console}`

File: distribution/docs/src/main/resources/content/_configuring/admin-console-tutorial.adoc [hunk 9]
```diff
@@ -62,7 +37,7 @@ Either view has an `>` arrow icon to view more information about the application
 |===
 
 .Application Detailed View
-Each individual application has a detailed view to view information specific to that application, adjust configurations or enable/disable features.
+Each individual application has a detailed view to adjust configurations specific to that application.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_configuring/admin-console-tutorial.adoc]: maybe `modify` configurations is better

File: distribution/docs/src/main/resources/content/_configuring/admin-console-tutorial.adoc [hunk 10]
```diff
@@ -92,9 +61,9 @@ Features are collections of OSGi bundles, configuration data, and/or other featu
 Features may have dependencies on other features and will auto-install them as needed.
 ====
 
-In the ${admin-console}, Features are found on the *Features* tab of each application.
+In the ${admin-console}, Features are found on the *Features* tab of the *System* tab.
 
-. Select the appropriate application.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_configuring/admin-console-tutorial.adoc]: same

File: distribution/docs/src/main/resources/content/_configuring/catalog-provider.adoc [hunk 11]
```diff
@@ -12,13 +12,12 @@ This scenario describes how to reconfigure ${branding} to use a different catalo
 This scenario assumes ${branding} is already running.
 
 .Uninstall Catalog Provider (if installed).
-. Navigate to the *${admin-console}*.
-. Select the *${ddf-solr}* application.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_configuring/catalog-provider.adoc]: same

File: distribution/docs/src/main/resources/content/_configuring/configuring-idp.adoc [hunk 12]
```diff
@@ -10,8 +10,8 @@
 .Installing the IdP from the ${admin-console}
 Install the included IdP implementation.
 
-. Navigate to the *${admin-console}*.
-. Select *Features*.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_configuring/configuring-idp.adoc]: same

File: distribution/docs/src/main/resources/content/_configuring/configuring-sts.adoc [hunk 13]
```diff
@@ -17,8 +17,7 @@ The STS is installed by default in ${branding}.
 
 .Configuring STS
 . Verify that the `serverKeystores.jks` file in `${home_directory}/etc/keystores` trusts the hostnames used in your environment (the hostnames of LDAP, and any ${branding} users that make use of this STS server).
-. Navigate to the *${admin-console}*.
-. Select the *${ddf-security}* application.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_configuring/configuring-sts.adoc]: same

File: distribution/docs/src/main/resources/content/_configuring/connecting-to-external-idp.adoc [hunk 14]
```diff
@@ -13,7 +13,6 @@ See <<_configuring_idp_sp, Configuring Security IdP Service Provider>> for detai
 .Disable Included IdP
 Remove the included IdP.
 
-. Navigate to the *${admin-console}*.
-. Select the *${ddf-security}* application.
-. Select *Features*.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_configuring/connecting-to-external-idp.adoc]: same

File: distribution/docs/src/main/resources/content/_configuring/external-sts.adoc [hunk 15]
```diff
@@ -31,8 +31,7 @@ Configure the location and credentials for the STS.
 .Disable Security STS Client Configuration
 Disable the client configuration for the Security STS
 
-. Navigate to the *${admin-console}*.
-. Select the *${ddf-security}* application.
-. Select *Features*.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_configuring/external-sts.adoc]: same

File: distribution/docs/src/main/resources/content/_configuring/http-proxy.adoc [hunk 16]
```diff
@@ -10,8 +10,7 @@
 The `platform-http-proxy` feature proxies https to http for clients that cannot use HTTPS and should not have HTTP enabled for the entire container via the `etc/org.ops4j.pax.web.cfg` file.
 
 .Enabling the HTTP Proxy from the ${admin-console}
-. Navigate to the *${admin-console}*.
-. Select the *${ddf-platform}* application tile.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_configuring/http-proxy.adoc]: same

File: distribution/docs/src/main/resources/content/_configuring/no-idp.adoc [hunk 17]
```diff
@@ -9,8 +9,7 @@
 
 To configure ${branding} to not use an Identity Provider (IdP), simply disable the IdP feature.
 
-. Navigate to the *${admin-console}*.
-. Select the *${ddf-security}* application.
-. Select *Features*.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_configuring/no-idp.adoc]: same

File: distribution/docs/src/main/resources/content/_devComponents/custom-applications.adoc [hunk 18]
```diff
@@ -4,15 +4,12 @@
 :summary: Developing applications.
 :order: 01
 
-The ${branding} applications are comprised of components, packaged as Karaf features, which are collections of OSGi bundles.
-These features can be installed/uninstalled using the ${admin-console} or ${command-console}.
-${branding} applications also consist of one or more OSGi bundles and, possibly, supplemental external files.
-These applications are packaged as Karaf KAR files for easy download and installation.
-These applications can be stored on a file system or a Maven repository.
+The ${branding} applications are a collection of OSGi bundles that contain configurations to be displayed in the ${admin-console}.
+The applications are defined in the ${etc/application-definitions} directory. Any json file added to the ${etc/application-definitions} with name, description and bundleLocations keys will be displayed in the ${admin-console}. The application will only display if one of the bundles listed in the bundleLocations key is running or there is at least 1 plugin associated with that application running.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_devComponents/custom-applications.adoc]: ✏️ can you break this line at the end of each sentence?

File: distribution/docs/src/main/resources/content/_endpoints/compression-services.adoc [hunk 19]
```diff
@@ -20,9 +20,8 @@ The compression services are not installed by default with a standard installati
 
 To Install:
 
-* Navigate to ${admin-console}.
-* Select the *${ddf-platform} Application*.
-* Select the *Features* Tab.
+* Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_endpoints/compression-services.adoc]: same

File: distribution/docs/src/main/resources/content/_endpoints/ftp-endpoint.adoc [hunk 20]
```diff
@@ -14,9 +14,8 @@ The FTP Endpoint is not installed by default with a standard installation.
 
 To install:
 
-* Navigate to ${admin-console}.
-* Navigate to *${ddf-catalog}* application.
-* Navigate to *Features* tab.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_endpoints/ftp-endpoint.adoc]: same

File: distribution/docs/src/main/resources/content/_endpoints/wps-endpoint.adoc [hunk 21]
```diff
@@ -18,9 +18,8 @@ The WPS Endpoint is not installed by default with a standard installation.
 
 To install:
 
-* Navigate to ${admin-console}.
-* Navigate to *${ddf-spatial}* application.
-* Navigate to *Features* tab.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_endpoints/wps-endpoint.adoc]: same

File: distribution/docs/src/main/resources/content/_plugins/metacard-backup-filestorage.adoc [hunk 22]
```diff
@@ -11,9 +11,8 @@ The Metacard Backup File Storage Provider is a storage provider that will store
 
 To install the Metacard Backup File Storage Provider
 
-. Navigate to the *${admin-console}*.
-. Select ${ddf-catalog} application.
-. Select *Features* tab.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_plugins/metacard-backup-filestorage.adoc]: same

File: distribution/docs/src/main/resources/content/_securityFramework/security-ldap-embedded-server.adoc [hunk 23]
```diff
@@ -16,18 +16,16 @@ The embedded LDAP server is intended for testing purposes only and is not recomm
 
 The embedded LDAP server is not installed by default with a standard installation.
 
-. Navigate to the ${admin-console}.
-. Select *Manage* Applications.
-. Install the *OpenDJ Embedded* application.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_securityFramework/security-ldap-embedded-server.adoc]: same

File: distribution/docs/src/main/resources/content/_transformers/geojson-metacard-xformer.adoc [hunk 24]
```diff
@@ -13,8 +13,7 @@ The GeoJSON Metacard Transformer is not installed by default with a standard ins
 
 To install:
 
-. Navigate to the *${admin-console}*.
-. Select the *${ddf-catalog}* application.
+. Navigate to the *System* tab.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_transformers/geojson-metacard-xformer.adoc]: same

File: distribution/docs/src/main/resources/content/_appReferences/mg-admin.adoc (hunk 2 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_appReferences/mg-admin.adoc]: just realized this might be unclear. I just mean eliminate the "more"

**Comment by tbatie** [comment]
build now

**Comment by tbatie** [comment]
CI completed build, there was a Network Error that occurred at the end.

**Comment by ricklarsen** [comment]
build now

**Pull request merged**